server/api/controllers/question.js

```diff
@@ -0,0 +1,24 @@
+const { Question } = require("../models/question");
+
+exports.get_questions = (req, res) => {
```

Comment:
Keep the names short and sweet. As the method is already inside question controller, a simple name as `get` would be enough.